app/components/mobile-work-filter.js

```diff
@@ -47,13 +57,20 @@ export default Component.extend({
       return this.attrs.clearFilters();
     },
     toggleFilterBlock() {
+      let innerScrollingContainerClass = vudu(c).liquidInner;
+      let $scrollContainer = $(`.${innerScrollingContainerClass}`);
+      let bottom = $scrollContainer.prop('scrollHeight');
+
       if (get(this, 'showFilterBlock')) {
         if (get(this, 'isFiltered')) {
           this.attrs.clearFilters();
+          $scrollContainer.animate({ scrollTop: bottom });
           return set(this, 'showFilterBlock', false);
         }
+        $scrollContainer.animate({ scrollTop: bottom });
         return set(this, 'showFilterBlock', false);
       }
+      $scrollContainer.animate({ scrollTop: bottom });
```

Comment:
Can we use greensock here?
https://greensock.com/forums/topic/11237-simple-scrollto/